backend/curation/api/v1/curation/collections/common.py

```diff
@@ -65,7 +65,7 @@ def reshape_for_curation_api(
         collection_id = collection_version.collection_id
         collection_url = f"{get_collections_base_url()}/collections/{collection_id.id}"
         revision_of = None
-        if not user_info.is_user_owner_or_allowed(collection_version.owner):
+        if not user_info or user_info.is_user_owner_or_allowed(collection_version.owner):
```

Comment:
should be` if not user_info or not user_info.user_info.is_user_owner_or_allowed(collection_version.owner)`